crypto/evp/e_des.c

```diff
@@ -7,6 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
+/*
+ * SHA-1 low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
+#include "internal/deprecated.h"
```

Comment:
What is the SHA usage in this file that requires this? I couldn't find it.

Reply:
Me either :(

Reply:
I'll take it out and see which CI breaks....